python/openassetio/hostAPI/Manager.py

```diff
@@ -1041,38 +1041,45 @@ def register(self, targetEntityRefs, entityTraitsDatas, context):
     ## @{
 
     @auditApiCall("Manager")
-    def createContext(self, parent=None):
+    def createContext(self):
         """
         Creates a new Context for use with the manager.
 
         @warning Contexts should never be directly constructed, always
-        use this method to create a new one.
-
-        @param parent openassetio.Context If supplied, the new context
-        will clone the supplied Context, and the Manager will be given a
-        chance to migrate any meaningful state etc... This can be useful
-        when certain UI elements need to 'take a copy' of a context in
-        its current state.
+        use this method or @ref createChildContext to create a new one.
 
+        @see @ref createChildContext
         @see @fqref{Context} "Context"
         """
-        context = Context()
+        return  Context(
+            managerState = self.__impl.createState(self.__hostSession)
```

Comment:
[question] Interesting formatting of keyword arguments, [pep8](https://peps.python.org/pep-0008/) says

> Don’t use spaces around the = sign when used to indicate a keyword argument

Is this something Black does?

Reply:
Ahh nice, yes, it does. I'd not been running `black` yet as I can't easily get it to do range formatting... Updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/464/commits/c3f29d56068ff9820a8bad484fdf2f96ea0b3263).